tools/vscode/refresh_compdb.sh

```diff
@@ -2,8 +2,8 @@
 
 tools/proto_format/proto_format.sh fix
```

Comment:
Can we possible put this behind a guard of some type? Either a prompt or have some env variable that can disable it (perhaps even in the format script itself)? It takes forever and there are times in which I know that things are up to date and I don't need to run it?

Reply:
the script does check git history for api/ changes https://github.com/envoyproxy/envoy/blob/master/tools/proto_format/proto_format.sh#L13 and skip most of it automatically.

Reply:
and `FORCE_PROTO_FORMAT` is the env var to force it run all steps.

Reply:
The problem is it runs it if there have been any changes ever to the tracking branch, not if I have made any changes since the last run. I end up having to comment this line out. I would like to have some way of actually only running the formatter when I want to run it, but I can do a separate PR to add something like the inverse of FORCE_PROTO_FORMAT if you want.

Reply:
I see, added SKIP_PROTO_FORMAT here (off by default).